platform/website/src/routes/[username]/+page.svelte

```diff
@@ -23,30 +25,51 @@
 		localStorage.setItem("layout_chatroomCollapsed", JSON.stringify(chatCollapsed));
 	}
 
-	$: viewers =
-		typeof data.user.channel.liveViewerCount === "number"
-			? viewersToString(data.user.channel.liveViewerCount)
-			: undefined;
-
 	let timeLive =
 		data.user.channel.lastLiveAt && formatDuration(new Date(data.user.channel.lastLiveAt));
-	let timeInterval: NodeJS.Timeout | number;
+	let viewers: number | undefined = undefined;
```

Comment:
why are you defining it with undefined instead of null?

Reply:
I'll change it FeelsOkayMan